assets/i18n/en_US.json

```diff
@@ -171,6 +178,7 @@
         "sourcesResetDialogTitle": "Reset",
         "sourcesResetDialogText": "Are you sure you want to reset custom sources to their default values?",
         "apiURLResetDialogText": "Are you sure you want to reset API URL to its default value?",
+        "sourcesUpdateNote": "Note: ReVanced Patches will be updated to the latest version automatically.",
```

Comment:
>When changing sources, the warning dialog may be show again, because the IP will be revealed to it.

@oSumAtrIX How about merging the warning text here?

Reply:
Can you clarify what you mean by that?

Reply:
Instead of another dialog
<img width="287" alt="image" src="https://github.com/ReVanced/revanced-manager/assets/39409020/ba0e4caa-323b-4a63-8f82-c6d43bd7a30b">

Reply:
Works